src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -139,11 +139,17 @@ public int MaxModelValidationErrors
 
         /// <summary>
         /// Gets or sets the flag which causes content negotiation to ignore Accept header 
-        /// when it contains the media type */*. <see langword="false"/> by default.
+        /// when it contains the media type */*. <c>false</c>c> by default.
         /// </summary>
         public bool RespectBrowserAcceptHeader { get; set; }
 
         /// <summary>
+        /// Gets or sets a flag to enable content-negotiation to select a formatter based on object type.
```

Comment:
Might want to add the *that is going to be serialized* bit to the comment

> formatter based on the object type being serialized.

or something along those lines.